app/controllers/Streamer.scala

```diff
@@ -27,6 +28,32 @@ final class Streamer(
       }
     }
 
+  def featured =
+    Open { implicit ctx =>
+      negotiate(
+        html = notFound,
+        api = _ =>
+          ctx.noKid ?? {
+            env.streamer.liveStreamApi.all
+              .map { streams =>
+                val max      = env.streamer.homepageMaxSetting.get()
+                val featured = streams.homepage(max, ctx.req, ctx.me) withTitles env.user.lightUserApi
+                Ok(
+                  Json.toJson(
+                    featured.live.streams.map(s =>
+                      Json.obj(
+                        "link"              -> routes.Streamer.redirect(s.streamer.id.value).absoluteURL(),
+                        "usernameWithTitle" -> featured.titleName(s),
+                        "status"            -> s.status
+                      )
```

Comment:
Can this be abstracted into a model?

Reply:
It can but I'm not sure it's useful since this model won't be reused somewhere else.

Reply:
Understood